files/moci-device-quarantine.sh

```
@@ -0,0 +1,194 @@
+	uci set firewall."$sid".src="lan"
+	uci set firewall."$sid".dest="$dest"
+	uci set firewall."$sid".src_mac="$mac"
+	uci set firewall."$sid".proto="all"
+	uci set firewall."$sid".target="REJECT"
+	uci set firewall."$sid".family="any"
+	uci set firewall."$sid".enabled="1"
+	return 0
+}
+
+quarantine_new_device() {
+	local mac="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)